relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -429,6 +429,10 @@ impl DialectWithParameters {
 }
 
 impl Dialect for DialectWithParameters {
+    fn dialect(&self) -> std::any::TypeId {
+        self.0.dialect()
+    }
```

Comment:
This informs the parser to treat the current dialect the same as the wrapped dialect, for example MySQL.